tests/emukit/conftest.py

```diff
@@ -0,0 +1,38 @@
+import GPy
```

Comment:
Can you add a comment on top of this file explaining what is it for, what does it contain?